src/qmapshack/main.cpp

```diff
@@ -30,6 +30,9 @@ int main(int argc, char ** argv)
 {
     QApplication app(argc, argv);
 
+    QApplication::setAttribute(Qt::AA_UseHighDpiPixmaps);
+    QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
+
```

Comment:
Those two lines have to go to the qmaptools binary, too.

Reply:
Done.